docs/logs/correlation/Program.cs

```diff
@@ -7,39 +7,39 @@
 using OpenTelemetry.Logs;
 using OpenTelemetry.Trace;
 
-namespace Correlation;
-
 public class Program
 {
-    private static readonly ActivitySource MyActivitySource = new(
-        "MyCompany.MyProduct.MyLibrary");
+    private static readonly ActivitySource MyActivitySource = new("MyCompany.MyProduct.MyLibrary");
 
     public static void Main()
     {
-        // Setup Logging
-        using var loggerFactory = LoggerFactory.Create(builder =>
+        var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
@reyang 

Just thinking about cross-cutting extensions and future plans. How would you personally feel if you could write this example like this?

```csharp
    public static void Main()
    {
        var sdk = OpenTelemetrySdk.Create(builder => builder
            .AddConsoleExporter()
            .WithTracing(tracing => tracing.AddSource("MyCompany.MyProduct.MyLibrary"))
            .WithLogging());

        var logger = sdk.GetLoggerFactory().CreateLogger<Program>();

        using (var activity = MyActivitySource.StartActivity("SayHello"))
        {
            // Write a log within the context of an activity
            logger.FoodPriceChanged("artichoke", 9.99);
        }

        // Dispose OpenTelemetry SDK before the application ends.
        // This will flush the remaining spans & logs and shutdown pipelines.
        sdk.Dispose();
    }
```

Reply:
Looks great to me 👍